app/views/handlers/__init__.py

```diff
@@ -8,4 +14,23 @@
     "IndividualResponseHandler",
     "IndividualResponseHowHandler",
     "IndividualResponsePostAddressConfirmHandler",
+    "individual_response_url",
 ]
+
+
+def individual_response_url(
+    schema: QuestionnaireSchema,
+    list_item_id: str,
+    questionnaire_store: QuestionnaireStore,
+    journey: str = None,
+) -> Union[str, None]:
+    if "individual_response" in schema.json:
+        for_list = schema.json["individual_response"]["for_list"]
```

Comment:
Given schema is only used for 'for_list', it would make sense to only pass its value. The recent merge to master includes a method to fetch this from schema now.